### PR TITLE
Week6 Homework

### PR DESCRIPTION
**Pull request by anastasiya-kliushkina**
Added cold flow for CountryInfoList screen
Added StateFlow

### CODE REVIEW

File: app/src/main/java/com/kodeco/android/countryinfo/flows/Flows.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    private val _backFlow = MutableStateFlow(0)
+    val backFlow = _backFlow.asStateFlow()
+
+    private val _counterFlow = MutableStateFlow(0)
+    val counterFlow = _counterFlow.asStateFlow()
+
+    init {
+        // Launch a coroutine to update counterFlow every second
+        GlobalScope.launch {
+            delay(1000) // delay on update start
```
Comment (ansh94) [inline app/src/main/java/com/kodeco/android/countryinfo/flows/Flows.kt:23]: Minor: To specify the Long value explicitly for delay, you can append the suffix L to the value. For eg 1000L or 1_000L

**Review by ansh94 — APPROVED** [review]
Nice job completing Week 6 homework assignment. All the flow objects are implemented correctly and the app is working as expected.

**Comment by anastasiya-kliushkina** [comment]
Thanks Anshdeep for your review! :)

File: app/src/main/java/com/kodeco/android/countryinfo/flows/Flows.kt (hunk 1 above)
Comment (anastasiya-kliushkina) [reply app/src/main/java/com/kodeco/android/countryinfo/flows/Flows.kt:23]: Thanks for pointing this out! Will update in my next PR

**Pull request merged**